src/dlob/dlob_node.rs

```diff
@@ -61,6 +61,12 @@ impl Eq for DirectionalNode {}
 
 impl PartialOrd for DirectionalNode {
     fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for DirectionalNode {
```

Comment:
Do all DLOB tests still pass with these changes?

Reply:
Yes, in my local,  the all test in dlob passed 

```
running 11 tests
test dlob::dlob_node::test::test_eq ... ok
test dlob::dlob_node::test::test_set_next_prev ... ok
test dlob::dlob_node::test::test_set_order ... ok
test dlob::dlob_node::test::test_vamm_node_get_user_account_panics - should panic ... ok
test dlob::dlob_node::test::test_vamm_node_set_order_panics - should panic ... ok
test dlob::order_list::tests::test_insertion_and_ordering ... ok
test dlob::tests::test_update_resting_limit_orders ... ok
test dlob::tests::test_get_resting_limit_bids ... ok
test dlob::tests::test_get_resting_limit_asks ... ok
test dlob::tests::test_dlob_insert ... ok
test dlob::tests::test_dlob_ordering ... ok

test result: ok. 11 passed; 0 failed; 0 ignored; 0 measured; 27 filtered out; finished in 0.00s

     Running tests/integration.rs (target/debug/deps/integration-3ef1abc8c0bfe3d8)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 4 filtered out; finished in 0.00s
```